src/licensedcode/plugin_license.py

```diff
@@ -18,6 +18,7 @@
 from commoncode.resource import clean_path
 from plugincode.scan import ScanPlugin
 from plugincode.scan import scan_impl
+import click
```

Comment:
This should be right after the `import attr` in the import order (and eventually we should use isort or usort to automate this)